src/js/components/TaskListComponent.jsx

```diff
@@ -167,10 +170,7 @@ var TaskListComponent = React.createClass({
                 </th>
             </tr>
           </thead>
-          <PagedContentComponent
```

Comment:
Excellent point @aldipower – this is much more performant! I think the PagedContentComponent is OK for the App Versions (those can only ever grow) but your solution is actually a much more scalable approach.